src/app/Http/Controllers/Operations/ShowOperation.php

```diff
@@ -78,6 +81,7 @@ public function show($id)
         // get the info for that entry (include softDeleted items if the trait is used)
         if ($this->crud->get('show.softDeletes') && in_array('Illuminate\Database\Eloquent\SoftDeletes', class_uses($this->crud->model))) {
             $this->data['entry'] = $this->crud->getModel()->withTrashed()->findOrFail($id);
+            $this->data['entry'] = $this->crud->setLocaleOnModel($this->data['entry'], $this->crud->get('show.useFallbackLocale'));
```

Comment:
Same here. Why does this PR make changes to the ShowOperation?

Reply:
The exact same reason I explained above ... but your call 👍